File: chatwoot/php/src/DeleteACustomFilterExample.php
<?php

namespace OSEG\ChatwootExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use Chatwoot;

$config = Chatwoot\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("userApiKey", "USER_API_KEY");
// $config->setApiKey("agentBotApiKey", "AGENT_BOT_API_KEY");
// $config->setApiKey("platformAppApiKey", "PLATFORM_APP_API_KEY");

try {
    (new Chatwoot\Client\Api\CustomFiltersApi(config: $config))->deleteACustomFilter(
        account_id: null,
        custom_filter_id: null,
    );
} catch (Chatwoot\Client\ApiException $e) {
    echo "Exception when calling CustomFiltersApi#deleteACustomFilter: {$e->getMessage()}";
}

File: chatwoot/ruby/src/DeleteACustomFilterExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
    config.api_key["api_access_token"] = "USER_API_KEY"
    # config.api_key["api_access_token"] = "AGENT_BOT_API_KEY"
    # config.api_key["api_access_token"] = "PLATFORM_APP_API_KEY"
end

begin
    ChatwootClient::CustomFiltersApi.new.delete_a_custom_filter(
        0, # account_id
        0, # custom_filter_id
    )
rescue ChatwootClient::ApiError => e
    puts "Exception when calling CustomFiltersApi#delete_a_custom_filter: #{e}"
end
